### ISSUE TITLE
NTR: Liver biopsy: vascular obliteration (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9158]
New term request Liver biopsy: vascular obliteration (elthat_100623003736): Occlusion; hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal
I wonder if we can make this more concrete, e.g., venous or arterial and any particular location?

**Comment by MickeySegal** [comment]
I changed it to be venous, as in https://www.ncbi.nlm.nih.gov/books/NBK1271/

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000716
- label: Terminal hepatic lobular vascular occlusion
- parents: [Abnormality of the hepatic vasculature (HP:0006707)](https://hpo.jax.org/app/browse/term/HP:0006707)
- pmid: PMID:17461490|PMID:20301448
- def: A type of Hepatic venous outflow obstruction characterized by fibrous concentric narrowing of zone 3 terminal hepatic venules, centrilobular hepatocyte necrosis, and sinusoidal congestion, accomapnied by necrosis of pericentral hepatocytes, and narrowing and eventual fibrosis of central veins.
- comment: Hepatic veno-occlusive disease (VOD) may present with fragmented red blood cells, fibrinogen, and Factor VIII/von Willebrand factor can be demonstrated in the subendothelial space of central veins and perivenular zones of hepatic acini. In the later stages, sinusoidal and venous lumen become obliterated by typeI, III, and IV collagen accompanied by an increase in the stellate cells that line the sinusoids. In some cases, fibrous bridges between central venules are observed.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9158
- synonyms: Hepatic sinusoidal obstruction syndrome(NONE)|Hepatic veno-occlusive disease(NONE)
